kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/status/NMDbusConnectorSignalConverterTest.java

```diff
@@ -0,0 +1,297 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.nm.status;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.experimental.runners.Enclosed;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Enclosed.class)
+public class NMDbusConnectorSignalConverterTest {
```

Comment:
The name of the class is misleading to me. You're testing `NMStatusConverter` methods here.

I would call this class `NMStatusConverterTest` and it will later be used to store all the tests for the `NMStatusConverter` class.

If you want to keep these tests separate from the other I would still rename this class `NMStatusConverterSignalConverterTest`

Reply:
Yeah... I'd keep these tests separated because of the parametric testing.